python-regression/tests/features/machine3/3_transaction_tests.feature.disabled

```diff
@@ -0,0 +1,75 @@
+        |keys                   |values                     |type           |
+        |transactions           |previousTransaction        |responseValue  |
+
+        #Give the node time to solidify the milestone
+        And we wait "15" second/seconds
+        
+        Given "getBalances" is called on "nodeA-m3" with:
+        |keys                   |values                     |type           |
+        |addresses              |TEST_EMPTY_ADDRESS         |staticList     |
+
+        Then the response for "getBalances" should return with:
+        |keys                   |values                     |type           |
+        |balances               |0                          |int            |  
+
+
+    Scenario: Incomplete bundle doesnt affect ledger state
+        We want to ascertain that ledger state is always calculated correctly.
+        Even in the presence of incomplete bundles being approved.
+
+        Then "1" transaction is issued on "nodeA-m3" with:
+        |keys                   |values                     |type           |
+        |address                |TEST_ADDRESS               |staticValue    |
+        |value                  |0                          |int            |
+        |tag                    |ZERO9VALUE                 |string         |
+
+        Then an incomplete bundle is generated referencing the previous transaction with:
+        |keys                   |values                     |type           |
+        |address                |TEST_EMPTY_ADDRESS         |staticValue    |
+        |tag                    |INCOMPLETE9TAG             |string         |
+
+        Then a transaction is issued referencing the previous transaction
+        |keys                   |values                     |type           |
+        |seed                   |THE_BANK                   |staticList     |
+        |address                |TEST_ADDRESS               |staticValue    |
+        |value                  |11                         |int            |
+        |tag                    |VALUE9TRANSACTION          |string         |
+
+        #In the default test, the latest sent index will be 53. The next milestone issued should be 54.
+        When a milestone is issued with index 54 and references:
+        |keys                   |values                     |type           |
+        |transactions           |previousTransaction        |responseValue  |
+
+        #Give the node time to solidify the milestone
+        And we wait "10" second/seconds
+        
+        Given "getBalances" is called on "nodeA-m3" with:
+        |keys                   |values                     |type           |
+        |addresses              |TEST_EMPTY_ADDRESS         |staticList     |
+
+        Then the response for "getBalances" should return with:
+        |keys                   |values                     |type           |
+        |balances               |0                          |int            | 
```

Comment:
can you also move this to a different PR?
Less chances we will forget about this and it will also help @acha-bill ...